src/main/java/Ultron/Commands/FindCommand.java

```diff
@@ -0,0 +1,54 @@
+package ultron.commands;
+
+import ultron.Storage;
+import ultron.TaskList;
+import ultron.exceptions.ExceptionType;
+import ultron.exceptions.UltronException;
+import ultron.tasks.Task;
+import ultron.ui.UI;
+
+public class FindCommand extends Command {
+
+    /**
+     * The find command for search function.
+     *
+     * @param arguments String to find.
+     */
+    public FindCommand(final String arguments) {
+        super(false, arguments);
+    }
+
+    /**
+     * Execute the command of Find command.
+     *
+     * @param taskList List of tasks.
+     * @param ui       UI for Ultron.
+     * @param storage  Storage for Ultron.
+     * @throws UltronException When there are no arguments.
+     */
+    @Override
+    public void execute(final TaskList taskList,
+                        final UI ui,
+                        final Storage storage) throws UltronException {
+
+        if (getArguments().length() == 0) {
+            throw new UltronException("find", getArguments(), ExceptionType.NO_ARGUMENTS_SUPPLIED);
+        }
+        boolean printed = false;
```

Comment:
Consider changing to `isPrinted`

Reply:
Thanks for the advice. Now it sounds more like a boolean name.